### DISCUSSION TITLE
create plugin for vscode, android studio

### DISCUSSION THREAD

**Discussion by Serge1996** [serverpod/serverpod#231] [Ideas]
create plugin for vscode, android studio

**Comment by vlidholt** [comment]
That is definitely in the plans. But focus right now is to get the version 1.0 out!

**Comment by yahu1031** [comment]
I think I can do this for ya... I did a basic ext for vscode.

**Reply by yahu1031** [reply]
[Serverpod VSCode Extension here](https://marketplace.visualstudio.com/items?itemName=minnu1031.serverpod)

**Comment by brajas-quinnox** [comment]
@Isakdl Will there be creating a plugin for android studio as well? https://github.com/serverpod/serverpod/pull/1010

**Reply by vlidholt** [reply]
@brajas-quinnox, we are building a Serverpod language server, which uses a standard format. Building plug-ins for different IDEs should be relatively easy. We are starting with VSCode though as it is the most commonly used.

**Reply by brajas-quinnox** [reply]
Thanks @vlidholt . Maybe this will be of interest to you. https://github.com/orgs/VeryGoodOpenSource/projects/1/views/16
